webapp/src/ts/polyfills.ts

```diff
@@ -88,7 +88,14 @@ declare global {
     CHTCore: any;
     angular: any;
     EnketoForm:any;
+    _phdcChanges: { // Additional namespace
+      // Specify your own contact_types here
+      hierarchyDuplicatePrevention: Partial<{[key in 'person' | 'health_center']: Strategy;}>;
```

Comment:
@jkuester @ChinHairSaintClair Following on from my above comment:

> The way I see it there are two point of config -- the excel files and the configuration json.

We probably couldn't expect the user to do config here. I agree the type safety is lovely and I'm not sure whether it's possible to generate a type from the config, but if not I don't see how we can keep the type safety and still maintain the existing configuration contract.